app/helpers/documents_helper.rb

```diff
@@ -1,6 +1,6 @@
 module DocumentsHelper
 
-  def format_date date
+  def format_document_date date
```

Comment:
Use parenthesis around parameters in function definitions: https://github.com/bbatsov/ruby-style-guide#method-parens